lambdatimes/src/App.js

```diff
@@ -3,14 +3,30 @@ import TopBar from './components/TopBar';
 import Header from './components/Header';
 import Content from './components/Content/Content';
 
```

Comment:
you probably want the initial state of loggedIn to be set to false, that way you can set it up to have the prompt render first, and then validate.